packages/core/helper-plugin/src/utils/getYupInnerErrors.ts

```diff
@@ -0,0 +1,30 @@
+import type { TranslationMessage } from 'src/types/shared';
+import type { ValidationError } from 'yup';
+
+const extractValuesFromYupError = (
+  errorType: string | undefined,
+  errorParams: Record<string, any> | undefined
```

Comment:
Couldn't you just make them optional?

```suggestion
  errorType?: string,
  errorParams?: Record<string, any>
```

Reply:
While you're not wrong mark, i want to just take the moment to let you both know there's a difference between `?` and `undefined`.

take this type:

```ts
interface Props {
	label?: string | undefined
}
```

This means, you can either not pass it, pass a string OR pass undefined. IIRC removing the `?` means you have to pass something, but this could then be `string | undefined`. The inverse is true if you omitted the `undefined` and kept `?`. This is helpful in _some_ circumstances 😄

Reply:
I think a combination would be better? there's little difference for this circumstance

```ts
  errorType?: string | undefined,
  errorParams?: Record<string, any> | undefined
```